doc/rst/source/animations.rst

```diff
@@ -81,8 +81,7 @@ includes the animations) does not take excessive time.
 
   :doc:`/animations/anim14`
 
-.. toctree::
-   :hidden:
-   :glob:
+- .. youtube:: Wk58r72g_nk
+     :width: 100%
```

Comment:
The youtube link occurs before the file references, so `Wk58r72g_nk` needs to replace `9D1qANCoptE` on line 79 and the toctree in these lines should remain.
```suggestion
.. toctree::
   :hidden:
   :glob
```